lib/Doctrine/DBAL/Schema/Table.php

```diff
@@ -253,12 +253,12 @@ public function columnsAreIndexed(array $columnsNames)
     }
 
     /**
-     * @param mixed[][] $columnNames
-     * @param string    $indexName
-     * @param bool      $isUnique
-     * @param bool      $isPrimary
-     * @param string[]  $flags
-     * @param mixed[]   $options
+     * @param string[]|mixed[][] $columnNames
```

Comment:
This is probably not `mixed[][]`. To inspect that, I'd suggest digging into the `Index` constructor.

Reply:
If it's not `mixed[][]`, then the code handling `$indexColOptions` should be removed. Given that this is a private API, I wanted to make the annotations consistent with the code and then clean up the code in a minor release.

Reply:
Indeed, it's only `string[]` Judging by the four calls from the same class. Please revert back to `string[] $columnNames` and rework the `foreach` loop to:
```php
foreach ($columnNames as $columnName) {
}
```